draft-ietf-quic-http.md

```diff
@@ -1742,7 +1866,7 @@ assigned by IANA.
 
 --- back
 
-# Considerations for Transitioning from HTTP/2
+# Considerations for Transitioning from HTTP/2 {#h2-considerations}
 
 HTTP/3 is strongly informed by HTTP/2, and bears many similarities.  This
 section describes the approach taken to design HTTP/3, points out important
```

Comment:
Below "HTTP/3 departs from HTTP/2 primarily where necessary to
accommodate the differences in behavior between QUIC and TCP (lack of ordering,	accommodate the 
support for streams)."

I'd tweak this to be more along the lines of "We changed it when we thought a new design was better, with particular focus on taking advantage of QUIC's lack of head of line blocking" to reflect the state we're in today.  Don't take my suggestion verbatim, but I think that's the concept?

Reply:
Based on the evidence, the design changes seem to be motivated because we lost something we had, not because we are taking advantage of some new ability.

Reply:
We've gotten a little stretchy in places, but we've been explicitly trying to stay out of the "improve on HTTP/2 in ways unrelated to QUIC" game.  (Some fixes to the more egregious gaps in prioritization are the primary exception.)  I'm trying to reflect that aspiration, though you're correct that we might want to reflect the stretchiness a bit as well.

Reply:
How about "HTTP/3 departs from HTTP/2 where QUIC's behavior differences from TCP are either beneficial or need to be accommodated (absence of linear ordering, support for framing)."